metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -142,7 +142,7 @@ def _build_metrics_output_node(self, query_spec: MetricFlowQuerySpec) -> Compute
                 MeasureSpec(
                     element_name=x.element_name,
                 )
-                for x in metric.measure_names
+                for x in metric.measure_references
```

Comment:
line 274, 280, 282 recommend `measure_names` change to `measure_specs`.

line 326, should `measure_spec` be `measure_spec.element_name`?